docs/src/pages/[platform]/connected-components/authenticator/headless/advanced/useAuthenticator.react-native.mdx

```diff
@@ -0,0 +1,58 @@
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+
+## useAuthenticator Hook
+
+`@aws-amplify/ui-react-native` ships with `useAuthenticator` React hook that can be used to access, modify, and update Authenticator's auth state. To use them, first wrap your application with [`<Authenticator.Provider>`](#authenticator-provider):
+
+```tsx
+import { Authenticator } from '@aws-amplify/ui-react-native';
+
+export default () => (
+  <Authenticator.Provider>
+    <App />
+  </Authenticator.Provider>
+);
+```
+
+Then, you can use `useAuthenticator` on your App:
+
+```tsx
+import { useAuthenticator } from '@aws-amplify/ui-react-native';
+
+const App = () => {
+  const { user, signOut } = useAuthenticator((context) => [context.user]);
+  // ...
+};
+```
+
+## Authenticator Provider
+
+In advanced use cases where usage of the [`useAuthenticator` hook](headless#useauthenticator-hook) outside the scope of the [`Authenticator`](../authenticator) is needed, wrap your application inside an `Authenticator.Provider`. The `Authenticator.Provider` guarantees that the [useAuthenticator hook](headless#useauthenticator-hook) is available throughout your application. You can see an example of this pattern in the [Protected Routes Guide.](../../guides/auth-protected)
+
+```jsx
+import { Authenticator } from '@aws-amplify/ui-react-native';
+import { View } from 'react-native';
```

Comment:
Not sure if this how all our docs are but generally think it's beet to place React and React Native imports above ours:
```suggestion
import { Text, View } from 'react-native';
import { Authenticator } from '@aws-amplify/ui-react-native';
```